catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java

```diff
@@ -132,11 +132,16 @@ public CreateResponse create(
     CreateResponse createResponse = null;
     CreateStorageRequest createStorageRequest = null;
     CreateStorageResponse createStorageResponse;
+    CreateRequest createRequest = null;
+    Exception ingestError = null;
 
     streamCreateRequest =
         opsStorageSupport.prepareStorageRequest(
             streamCreateRequest, streamCreateRequest::getContentItems);
 
+    INGEST_LOGGER.info(
+        "Resource ingest has started for resource id: {}.", streamCreateRequest.getId());
```

Comment:
✏️ It'd be more valuable to admins/users if we logged the filename instead of the id.

Reply:
Agree with @brjeter. That is what an admin like Sal is looking for.